src/components/forms/ProjectGrantsForm.tsx

```diff
@@ -290,6 +311,13 @@ export const ProjectGrantsForm: FC = () => {
                 </PageText>
               </Box>
             )}
+            {errors?.company?.type === 'validate' && (
+              <Box mt={1}>
+                <PageText as='small' fontSize='helpText' color='red.500'>
+                  Organization name contain a URL.
```

Comment:
For consistency within the error messages and more clarity, we should rephrase it as 'Organization name cannot contain a URL'